.github/workflows/docker.yml

```diff
@@ -48,9 +48,18 @@ jobs:
           echo "version=${VERSION}" >> $GITHUB_OUTPUT
           echo "tags=${TAGS}" >> $GITHUB_OUTPUT
           echo "created=$(date -u +'%Y-%m-%dT%H:%M:%SZ')" >> $GITHUB_OUTPUT
-
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
+          
+          # Determine the platforms based on the branch
+          PLATFORMS="linux/amd64"
+          REF_NAME=$(echo "${GITHUB_REF#refs/*/}")
+          if [[ $GITHUB_REF == refs/tags/* ]]; then
+            PLATFORMS="linux/amd64,linux/arm64"
```

Comment:
instead of having two if/else with the same body, could use the bash's OR syntax:
https://unix.stackexchange.com/a/47585/324980